packages/core/src/sass/account-wizard.scss

```diff
@@ -189,30 +189,18 @@
                 margin-top: 1.6rem;
             }
         }
-        // fix for safari bug with autoheight wrapper not working properly for calculating modal footer offset
-        @media not all and (min-resolution: 0.001dpcm) {
-            // stylelint-disable-line
-            @supports (-webkit-appearance: none) and (stroke-color: transparent) {
-                @include desktop {
-                    max-height: calc(100% - 48px);
-                }
-                @include tablet {
-                    max-height: calc(100% - 48px);
-                }
-            }
-        }
         @include desktop {
-            height: 100%;
             display: grid;
             grid-template-rows: minmax(20rem, 1fr);
         }
         @include tablet {
-            height: 100%;
             display: grid;
             grid-template-rows: minmax(20rem, 1fr);
         }
     }
     &__body {
+        display: grid;
```

Comment:
any reason why we use display grid without template here instead of flex?

Reply:
Actually the result will be the same. I used "grid" depend on rest of the code. It seams "grid" is the developer's choice for responsive behavior instead of flex. I prefer "flex".

Reply:
if possible can use flex here 🙏 . we use grids normally for items that is sorted in a grid, for wrapper or container, its better to use flex

Reply:
Yes sure. It's better in my opinion too. 😊